crates/frontend/src/components/default_config_form.rs

```diff
@@ -141,7 +141,7 @@ where
                     type="text"
                     placeholder="Key"
                     class="input input-bordered w-full max-w-md"
-                    value=config_key.get()
+                    value=config_key.get_untracked()
```

Comment:
Are we sure get_untracked is right here? Do we not want reactivity?

Reply:
it was anyways not reactive earlier at this point, this just makes it explicit and avoids warnings